src/test/java/core/basesyntax/CalculatorTest.java

```diff
@@ -0,0 +1,165 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class CalculatorTest {
+    private static final double DELTA = 0.0001;
+    private static Calculator calculator;
+    private double expected;
+    private double actual;
+
+    @BeforeAll
+    static void beforeAll() {
+        calculator = new Calculator();
+    }
+
+    @Test
+    void calculate_additionTwoPositiveOperands_Ok() {
+        expected = 15d + 20d;
```

Comment:
we can set the result value for a variable without adding two values (this comment is optional)
```suggestion
        expected = 35d;
```